### PR TITLE
Switching over to use new version service functions

### PR DESCRIPTION
**Pull request by jason-rodri**
Moving version service to new functions and cleaning up

### CODE REVIEW

File: crypto/fips/fips_post.c [hunk 1]
```diff
@@ -70,10 +70,6 @@
 # include "crypto/rand.h"
 # include "fips_locl.h"
 
-char* FIPS_show_version(void)
-{
-    return "Rocky Linux 8 OpenSSL Cryptographic Module Version Rocky8.20240321";
```
Comment (jason-rodri) [inline crypto/fips/fips_post.c:75]: Do we just revert the commit that added these functions?

Comment (jallisonciq) [reply crypto/fips/fips_post.c:75]: Seems so. Sorry.

**Comment by jallisonciq** [comment]
LGTM !

**Review by jallisonciq — APPROVED** [review]
Please push !

**Pull request merged**